source/deployment/sso-saml-adfs-msws2016.rst

```diff
@@ -0,0 +1,208 @@
+ - An SSL certificate to sign your ADFS login page.
+ - ADFS installed on your Microsoft Server. You can find a detailed guide for deploying and configuring ADFS in `this article <https://msdn.microsoft.com/en-us/library/gg188612.aspx>`__.
+
+On your ADFS installation, note down the value of the **SAML 2.0/W-Federation URL** in ADFS Endpoints section, also known as the **SAML SSO URL Endpoint** in this guide. If you chose the defaults for the installation, this will be ``/adfs/ls/``.
+
+Add a Relying Party Trust
+-------------------------
+
+1. In ADFS management snap-in, go to **AD FS > Relying Party Trusts** and click **Add Relying Party Trust** on the right sidebar. As an alternative right-click **Relying Party Trusts** and choose **Add Relying Party Trust** from the context menu.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_000.png
+
+2. A configuration wizard for adding a new relying party trust opens. In the **Welcome** screen, choose **Claims aware** and click **Start**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_001.png
+
+3. In the **Select Data Source** screen, select the option **Enter data about the relying party manually**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_002.png
+
+4. In the **Specify Display Name** screen, enter a **Display Name** to recognize the trust, such as ``Mattermost``, and add any notes you want to make.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_003.png
+
+5. In the **Configure Certificate** screen, leave the certificate settings at their default values.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_004.png
+
+However, if you would like to set up encryption for your SAML connection, click the **Browse** button and upload your Service Provider Public Certificate.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_005.png
+
+6. In the **Configure URL** screen, select the option **Enable Support for the SAML 2.0 WebSSO protocol** and enter the **SAML 2.0 SSO service URL**, which is of the form ``https://<your-mattermost-url>/login/sso/saml`` where ``https://<your-mattermost-url>`` should typically match the `Mattermost Site URL <https://docs.mattermost.com/administration/config-settings.html#site-url>`__.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_006.png
+
+7. In the **Configure Identifiers** screen, enter the **Relying party trust identifier** (also known as the **Identity Provider Issuer URL**) of the form ``https://<your-idp-url>/adfs/services/trust`` and click **Add**.
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_007.png
+
+	.. image:: ../../source/images/SSO-SAML-ADFS_add-new-relying-party-trust_008.png
+
+Add your **SAML 2.0 SSO service URL** from above the same way.
```

Comment:
Is this added to the "Configure Identifiers" menu as a separate entry, or is it in the same process?

Reply:
The process is identical to the one above using a different URL, resulting in two separate entries in the "Configure Identifiers" step.